<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1309</width>
    <height>636</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="sizeConstraint">
    <enum>QLayout::SetNoConstraint</enum>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item row="0" column="1">
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabAll">
      <attribute name="title">
       <string>All</string>
      </attribute>
      <widget class="QTextEdit" name="textEdit">
       <property name="geometry">
        <rect>
         <x>635</x>
         <y>27</y>
         <width>751</width>
         <height>581</height>
        </rect>
       </property>
      </widget>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>149</x>
         <y>30</y>
         <width>151</width>
         <height>30</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_serialNumber">
        <item>
         <widget class="QLabel" name="label_serialNumber">
          <property name="text">
           <string>Serial Number</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit_serialNumber"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_2">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>30</y>
         <width>171</width>
         <height>25</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="label_purchaseDate">
          <property name="text">
           <string>Date of Purchase</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="dateEdit_purchaseDate"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_3">
       <property name="geometry">
        <rect>
         <x>492</x>
         <y>30</y>
         <width>131</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_model">
        <item>
         <widget class="QLabel" name="label_model">
          <property name="text">
           <string>Model</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_model"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_4">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>121</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_status">
        <item>
         <widget class="QLabel" name="label_status">
          <property name="text">
           <string>Status</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_status"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_5">
       <property name="geometry">
        <rect>
         <x>310</x>
         <y>70</y>
         <width>171</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_country">
        <item>
         <widget class="QLabel" name="label_country">
          <property name="text">
           <string>Country </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_country"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_6">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>30</y>
         <width>121</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_type">
        <item>
         <widget class="QLabel" name="label_type">
          <property name="text">
           <string>Type </string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_type"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_7">
       <property name="geometry">
        <rect>
         <x>149</x>
         <y>70</y>
         <width>151</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_instytution">
        <item>
         <widget class="QLabel" name="label_institution">
          <property name="text">
           <string>Institution</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_institution"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="layoutWidget_8">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>70</y>
         <width>131</width>
         <height>24</height>
        </rect>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_room">
        <item>
         <widget class="QLabel" name="label_room">
          <property name="text">
           <string>Room No.</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboBox_room"/>
        </item>
       </layout>
      </widget>
      <widget class="QTableView" name="tableView_all">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>130</y>
         <width>621</width>
         <height>451</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_search">
       <property name="geometry">
        <rect>
         <x>490</x>
         <y>100</y>
         <width>131</width>
         <height>22</height>
        </rect>
       </property>
       <property name="text">
        <string>Search</string>
       </property>
      </widget>
     </widget>
     <widget class="QWidget" name="tab_sipm">
      <attribute name="title">
       <string>SiPM</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_afe">
      <attribute name="title">
       <string>AFE</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_hub">
      <attribute name="title">
       <string>Hub</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_scintillator">
      <attribute name="title">
       <string>Scintillator</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_detector">
      <attribute name="title">
       <string>Detector</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_section">
      <attribute name="title">
       <string>Section</string>
      </attribute>
     </widget>
     <widget class="QWidget" name="tab_module">
      <attribute name="title">
       <string>Module</string>
      </attribute>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
